.typos.toml

```diff
@@ -0,0 +1,21 @@
+[default]
+extend-ignore-re = [
+    "\\b[0-9A-Za-z+/]{91}(=|==)?\\b",  # base64 strings
+    "[0-9a-fA-F]{7,}",                 # git commit hashes
+    "\\b[0-9A-Za-z+/]{33,}(=|==)?\\b", # SHA/tpub/adresses etc strings
```

Comment:
In 81bbee2cb82a54c81dbe7b8f60042c8cbf61b6f7:

Isn't basically every single English word caught by at least one of these exclusions?

Reply:
I'll remove it and check what it catches.

Reply:
Removed, it still works.
Then let's keep it without it.